### DISCUSSION TITLE
Possible to access multiple media collections?

### DISCUSSION THREAD

**Discussion by jakehopking** [payloadcms/payload#1146] [Roadmap]
Hi - is it possible to select/upload images from/to multiple media collections?

It's possible to select multiple collections from within the richtext editor:

<img width="656" alt="Screenshot 2022-09-19 at 22 59 47" src="https://user-images.githubusercontent.com/3432643/191128169-4b044827-cbd2-452f-bc4b-52edd65c224e.png">

However I can't see how it's possible to reference an array of media collections elsewhere. This kind of makes sense from a simplistic upload perspective, however it would be nice to be able to select a collection to upload to, as well as selecting images from different collections of existing images.

<img width="843" alt="Screenshot 2022-09-19 at 23 04 44" src="https://user-images.githubusercontent.com/3432643/191128333-947ab2f7-43fa-44e5-9170-6b88cff8886a.png">

```typescript
{
  name: 'heroMedia',
  label: 'Hero media',
  labels: {
    singular: 'Hero media',
    plural: 'Hero media',
  },
  type: 'array',
  minRows: 1,
  maxRows: 2,
  fields: [
    {
      name: 'media',
      label: 'Media',
      relationTo: [MediaSlugs.media, MediaSlugs.mediaPhotos], // <== how can we achieve this?
      type: 'upload',
      required: true,
    },
  ],
},
```

Thanks 🙏

**Comment by jmikrut** [comment]
Hey @jakehopking this is a great idea. We have also been asked to support `hasMany: true` within our upload fields, and that is also something we are considering. 

I am going to tag this as a Feature Request and set it to `planned`. 

😎

**Reply by jakehopking** [reply]
Thanks for the reply. Look forward to this featuring!
Until then, I guess I could use a radio type and an admin condition to render either media collections in each array item?

**Comment by immotus** [comment]
This is now covered by the https://github.com/payloadcms/payload/pull/14363
